### PR TITLE
feat: update layout for algolia search

### PR DESCRIPTION
**Pull request by itsacoyote**
# Description

Update page layout for algolia search crawler.

### CODE REVIEW

**Review by jennyg0 — APPROVED** [review]

**Review by jennyg0 — COMMENTED** [review]
oh there is still a lint check failing

File: content/tutorials/native-aa-multisig/10.index.md [hunk 1]
```diff
@@ -964,8 +964,8 @@ export default async function (hre: HardhatRuntimeEnvironment) {
 
 ::callout{icon="i-heroicons-light-bulb"}
 ZKsync has different address derivation rules from Ethereum.
-Always use the [`createAddress`](https://sdk.zksync.io/js/ethers/api/v6/utilities/utils#createaddress)
-and [`create2Address`](https://sdk.zksync.io/js/ethers/api/v6/utilities/utils#create2address) utility functions of the `zksync-ethers` SDK.
+Always use the [`createAddress`](https://sdk.zksync.io/js/ethers/api/v6/utilities#createaddress)
+and [`create2Address`](https://sdk.zksync.io/js/ethers/api/v6/utilities#create2address) utility functions of the `zksync-ethers` SDK.
```
Comment (jennyg0) [inline content/tutorials/native-aa-multisig/10.index.md:968]: https://sdk.zksync.io/js/ethers/api/v6/utilities/utils#createaddress 
https://sdk.zksync.io/js/ethers/api/v6/utilities/utils#create2address

Comment (jennyg0) [reply content/tutorials/native-aa-multisig/10.index.md:968]: oh nvm these are updated in sdk pr?

**Review by jennyg0 — APPROVED** [review]

File: content/tutorials/native-aa-multisig/10.index.md (hunk 1 above)
Comment (itsacoyote) [reply content/tutorials/native-aa-multisig/10.index.md:968]: Yep, they're getting updated in the SDK PR. I'll merge that first before doing this and the ZKsync Docs one.

**Pull request merged**